File: pubfi/tmp/tmp.js
const requestBody = {
  "validityInMinutes": 123,
  "secret": "string"
};

fetch("https://api.public.com/userapiauthservice/personal/access-tokens", {
  method: "POST",
  headers: {
    "Content-Type": "application/json",
    "User-Agent": "public-dev-docs"
  },
  body: JSON.stringify(requestBody)
})
.then(response => response.json())
.then(data => console.log(data));

File: pubfi/tmp/tmp.py
import requests

url = "https://api.public.com/userapiauthservice/personal/access-tokens"
headers = {
    "Content-Type": "application/json",
    "User-Agent": "public-dev-docs"
}

request_body = {
  "validityInMinutes": 123,
  "secret": "string"
}

response = requests.post(url, headers=headers, json=request_body)
data = response.json()
print(data)
